docs/dev/api/api-testing.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: api-testing
-title: API Testing API Methods
+title: API Testing API Endpoints
 sidebar_label: API Testing
 ---
 
```

Comment:
You could also update the reference to "methods" in the description below (on line 10):

"These API _methods_ allow you to interact with Sauce Labs API Testing functionality. You can execute tests, view analytics, retrieve project information, and more."